src/components/molecules/VenuePartygoers/VenuePartygoers.tsx

```diff
@@ -1,31 +1,38 @@
 import React, { useMemo } from "react";
 
-import { useSelector } from "hooks/useSelector";
-import { usePartygoers } from "hooks/users";
-
-import { currentVenueSelector, parentVenueSelector } from "utils/selectors";
+import { useRecentWorldUsers } from "hooks/users";
+import { useConnectRelatedVenues } from "hooks/useConnectRelatedVenues";
+import { useVenueId } from "hooks/useVenueId";
 
 import "./VenuePartygoers.scss";
 
 export const VenuePartygoers = () => {
-  const venue = useSelector(currentVenueSelector);
-  const parentVenue = useSelector(parentVenueSelector);
-  const partygoers = usePartygoers();
-
-  const currentVenueTitle = venue?.attendeesTitle ?? "partygoers";
-  const attendeesTitle = parentVenue?.attendeesTitle ?? currentVenueTitle;
-  const venueName = venue?.name;
-
-  const currentVenuePartygoers = useMemo(() => {
-    if (!venueName) return [];
-
-    return partygoers.filter((partygoer) => partygoer.lastSeenIn?.[venueName]);
-  }, [partygoers, venueName]);
-  const numberOfPartygoers = currentVenuePartygoers.length;
```

Comment:
The logic from this isn't accurately reflected in the new code that uses `useRecentWorldUsers`.

Reply:
This is intentional. This is one of the places, where we need to use recentWorldUsers since this is this title on the top of the page that says: {NUMBER_OF_PARTYGOERS} partygoers online

Reply:
I think we need to display all world users currently online there

Reply:
Cool, yup, that makes sense.